docs/source/examples/example_twitch.rst

```diff
@@ -0,0 +1,320 @@
+we would like to assess the quality of the data in version 2.
+
+The original data set can be found on `kaggle <https://www.kaggle.com/datasets/aayushmishra1512/twitchdata>`_.
+For the sake of this tutorial, we slightly process it and provide two version of it.
+One can either recreate this by executing this `processing <https://github.com/Quantco/datajudge/tree/main/docs/source/examples/twitch_upload.py>`_
+oneself on the original data or download our processed files (`version 1 <https://github.com/Quantco/datajudge/tree/main/docs/source/examples/twitch_version1.csv>`_
+and `version 2 <https://github.com/Quantco/datajudge/tree/main/docs/source/examples/twitch_version2.csv>`_) right away.
+
+The aforementioned script also populates two tables in a Postgres database. This resembles the following:
+
+.. code-block:: python
+
+    address = os.environ.get("DB_ADDR", "localhost")
+    connection_string = f"postgresql://datajudge:datajudge@{address}:5432/datajudge"
+    engine = sa.create_engine(connection_string)
+    df_v2.to_sql("twitch_v2", engine, schema="public", if_exists="replace")
+    df_v1.to_sql("twitch_v1", engine, schema="public", if_exists="replace")
+
+
+Once the tables are stored in a database, we can actually write a ``Datajudge``
+specification against them. But first, we'll have a look at what the data roughly
+looks like by investigating a random sample of four rows:
+
+.. list-table:: A sample of the data
+   :header-rows: 1
+
+   * - channel
+     - watch_time
+     - stream_time
+     - peak_viewers
+     - average_viewers
+     - followers
+     - followers_gained
+     - views_gained
+     - partnered
+     - mature
+     - language
+   * - xQcOW
+     - 6196161750
+     - 215250
+     - 222720
+     - 27716
+     - 3246298
+     - 1734810
+     - 93036735
+     - True
+     - False
+     - English
+   * - summit1g
+     - 6091677300
+     - 211845
+     - 310998
+     - 25610
+     - 5310163
+     - 1374810
+     - 89705964
+     - True
+     - False
+     - English
+   * - Gaules
+     - 5644590915
+     - 515280
+     - 387315
+     - 10976
+     - 1767635
+     - 1023779
+     - 102611607
+     - True
+     - True
+     - Portuguese
+   * - ESL_CSGO
+     - 3970318140
+     - 517740
+     - 300575
+     - 7714
+     - 3944850
+     - 703986
+     - 106546942
+     - True
+     - False
+     - English
+
+Note that we expect both version 1 and version 2 to follow this structure. Due to them
+being assembled at different points in time, merely their rows shows differ.
+
+
+Now let's write an actual specification, expressing our expectations against the data.
+First, we need to make sure a connection to database can be established at test execution
+time. How this is done exactly depends on how you set up your database. When using our
+default setup with running `$ ./start_postgres.sh <https://github.com/Quantco/datajudge/blob/main/start_postgres.sh>`_,
```

Comment:
Probably here would be good to mention it's a container-based setup.

Reply:
Good point! I restructured this a bit to talk about the dockerized setup at the very beginning.